src/layout/__tests__/Body.test.js

```diff
@@ -0,0 +1,91 @@
+/*global describe, test, expect */
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { mapStateToProps } from '../Body';
+import Body from '../Body';
+import { ChartSpikeIcon } from '@patternfly/react-icons';
+
+const mockStore = configureMockStore();
+const store = mockStore({});
+
+const activeTechnologiesMock = [
+    {
+        entitlement: 'test',
+        marketing: true,
+        marketingImage: ChartSpikeIcon,
+        marketingText: 'Marketing Test',
+        marketingUrl: '/marketing-test',
+        name: 'test',
+        id: 'test',
+        url: 'test',
+        baseApp: '/test',
+        apps: {
+            rules: '/app-test'
+        },
+        icon: ChartSpikeIcon,
+        title: 'Jest Test',
+        emptyTitle: 'test',
+        emptyText: 'test',
+        emptyAction: {
+            title: 'test',
+            navigate: '/empty-action-test'
+        },
+        body: 'test'
+    }
+];
+
+function getInput(obj) {
+    return {
+        technologies: {
+            activeTechnologies: [ obj ]
+        }
+    };
+}
+
+function getOutput(obj) {
+    return { technologies: [ obj ]};
+}
+
+describe('mapStateToProps', () => {
+    test('should return the re-wrapped data', () => {
+        const data = { foo: 'bar' };
+        const output = mapStateToProps(getInput(data));
+        expect(output).toEqual(getOutput(data));
+    });
+
+    test('should filter out inactive', () => {
+        const data = {
+            foo: 'bar',
+            baz: { disabled: true }
+        };
+        const output = mapStateToProps(getInput(data));
+        expect(output).toEqual(getOutput(data));
+    });
+
+    test('should only filter out disabled:true', () => {
+        const data = {
+            foo: 'bar',
+            bar: { disabled: false },
+            baz: { disabled: true }
+        };
+        const output = mapStateToProps(getInput(data));
+        expect(output).toEqual(getOutput(data));
+    });
+});
+
+describe('render Body component', () => {
+    it('should render correctly', () => {
+        const wrapper = mount(
+            <Provider store={ store }>
+                <Router>
+                    <Body technologies={ activeTechnologiesMock }/>
```

Comment:
You shouldn't pass redux state as prop instead use mocked store. Also try to limit snapshots as much as possible by using [render](https://airbnb.io/enzyme/docs/api/ShallowWrapper/render.html), `find` selector or [shallowtojson](https://github.com/adriantoine/enzyme-to-json/blob/HEAD/docs/helper-functions.md#shallowtojson). Often times you don't really have to store the entire DOM tree and that might actually be problematic for us in future if PF changes some classnames or default props.
```JSX
const store = mockStore({technologies: activeTechnologiesMock });
const wrapper = mount(
<Provider store={ store }>
  <Body />
</Provider>
expect(toJson(wrapper.find('Body'))).toMatchSnapshot();
);
```